tests/Support/NovaTestMockTrait.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace RonasIT\Support\Tests\Support;
+
+use Illuminate\Support\Facades\View;
+use Mockery;
+use Mockery\MockInterface;
+use org\bovigo\vfs\vfsStream;
+use phpmock\Mock;
+use phpmock\MockBuilder;
+use RonasIT\Support\Generators\NovaTestGenerator;
+
+trait NovaTestMockTrait
+{
+    public function mockNativeFunction(string $namespace, string $name, callable $callback): Mock
```

Comment:
let's get result argument instead of callback

```suggestion
    public function mockNativeFunction(string $namespace, string $name, $resulr): Mock
```